node/src/Transaction.ts

```diff
@@ -384,6 +387,41 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
         return this.addAndReturn(createDel(keys));
     }
 
+    /**
+     * Serialize the value stored at `key` in a Valkey-specific format and return it to the user.
+     *
+     * See https://valkey.io/commands/dump/ for details.
+     *
+     * @param key - The key of the set.
+     *
+     * Command Response - The serialized value of a set. If `key` does not exist, `null` will be returned.
+     */
+    public dump(key: GlideString): T {
+        return this.addAndReturn(createDump(key));
+    }
+
+    /**
+     * Create a `key` associated with a `value` that is obtained by deserializing the provided
+     * serialized `value` (obtained via {@link dump}).
+     *
+     * See https://valkey.io/commands/restore/ for details.
+     *
+     * @param key - The key of the set.
+     * @param ttl - The expiry time (in milliseconds). If `0`, the `key` will persist.
+     * @param value - The serialized value.
+     * @param options - (Optional) Restore options {@link RestoreOptions}.
+     *
+     * Command Response - Return "OK" if successfully create a `key` with a `value`.
+     */
+    public restore(
+        key: GlideString,
+        ttl: number,
+        value: GlideString,
```

Comment:
```suggestion
        value: Buffer,
```

Reply:
I am getting error with this change:

```
Argument of type 'protobuf.Buffer' is not assignable to parameter of type 'Buffer'.
  Type 'Uint8Array' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 66 more.ts(2345)
```

Reply:
Probably you need to import `Buffer` or import the right `Buffer` (not protobuf one)